docs/toc.yml

```diff
@@ -1,6 +1,4 @@
 - name: Home
-  href: index.md
-- name: Getting Started
```

Comment:
It's weird to me that the Home page should be the concepts page that points to the get started page. And how do you get back to the landing page?

Reply:
The idea is that the content in the get started page is the landing & home page.

Reply:
Its a little weird that the other docs aren't in the 'concepts' category, but I didn't want to break links so I left the file structure alone.

Reply:
I'm not following. This is the top menu for the entire site, and you removed an entry from the menu that brings you back to the landing page.
The getting-started sub section has its own toc.yaml file that controls the menu on the left.